app/themes/default/views/layouts/default.html.slim

```diff
@@ -33,7 +33,18 @@ html
         div.collapse.navbar-collapse#site-navigation-navbar
           ul.nav.navbar-nav.pull-right
             li
-              = link_to(t('layout.navbar.courses'), courses_path)
+              - my_courses = user_signed_in? && Course.containing_user(current_user).ordered_by_end_at
```

Comment:
Maybe this ought to be cached at view level; I've not completely figured out how to add caching to these things.